crates/rolldown/src/utils/render_ecma_module.rs

```diff
@@ -38,13 +38,13 @@ pub fn render_ecma_module(
             .with_pre_compute_sourcemap_data(options.is_sourcemap_enabled()),
         ));
       } else {
-        sources.push(Box::new(render_output.code));
+        sources.push(Box::new(render_output.code.trim_end().to_string()));
       }
     } else {
-      sources.push(Box::new(render_output.code));
+      sources.push(Box::new(render_output.code.trim_end().to_string()));
```

Comment:
The `to_string` allocate the code again at here. It is not worth to do it. 

Other: Here the line 37 also need to change, it need to make sure the sourcemap is valid.

Reply:
Thank you for your review. I am not familiar with Rust, how should I modify it here?

Reply:
> Thank you for your review. I am not familiar with Rust, how should I modify it here?
It need to update the code structure at here, the source need to owner the `String` and trim_end `&str`.

I think it is not worth doing it because it will increase the complex, just using the oxc codgen code is fine.

Reply:
You mean this modification is unnecessary and the code structure can remain as it is, right?

Reply:
Yes. But also thanks for your contribution.